src/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const [completed, setCompleted] = useState(todo.completed);
+
+  const completedDealToggler = () => {
```

Comment:
```suggestion
  const toggleCompleted = () => {
```
Get used to name functions according to these rules
![image](https://user-images.githubusercontent.com/96409563/228926389-329a0ae7-c194-47ee-85bc-791a35923db2.png)